cassandra/example/src/main/java/example/springdata/cassandra/basic/BasicUserRepository.java

```diff
@@ -23,4 +26,13 @@
  * 
  * @author Thomas Darimont
  */
-public interface SimpleUserRepository extends CrudRepository<User, Long> {}
+public interface BasicUserRepository extends CrudRepository<User, Long> {
```

Comment:
Maybe add some brief JavaDoc to the individual methods to explain, what exactly they're showing (i.e. what's the special thing about *this* method over the others).